codalab/server/worker_info_accessor.py

```diff
@@ -25,7 +26,7 @@ def worker_with_id(self, user_id, worker_id):
         return None
 
     def user_owned_workers(self, user_id):
-        return list(self._user_id_to_workers[user_id])
+        return list(copy.deepcopy(worker) for worker in self._user_id_to_workers[user_id])
```

Comment:
Can the importance of this deepcopy (what is mutable, etc.) be documented somewhere?

Reply:
@percyliang Just added.